ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```diff
@@ -130,6 +146,9 @@ fi
 
 if [ "${ADD_INGRESS_RECORDS_MANUALLY}" == "yes" ]; then
 
+  export KUBECONFIG="${dir}/auth/kubeconfig"
+  wait_router_lb_provision || exit 1
```

Comment:
@wking  while installing an private IPI cluster in a disconnected environment, `openshift-install create cluster` command [1] alway get failed, since we need to manually create apps DNS, and then `wait-for install-complete`,  this is why I removed `errexit` and `pipefail`, so I could control whether exit program or not when an error occurred

https://github.com/yunjiang29/release/blob/aws-sno/ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh#L138